index.html

```diff
@@ -3,9 +3,11 @@
 
 <head>
     <meta charset="UTF-8">
+    <meta name="description" content="Explore my personal portfolio of front-end development projects. Discover my work as a developer, including websites and web applications I've created.">
+    <meta name="author" content="Mikael Elias">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio</title>
-    <link rel="stylesheet" href="styles/index.css">
+    <title>About</title>
```

Comment:
Acho que vc pode colocar seu nome no titulo aqui.
```suggestion
    <title>Mikael Elias</title>
```

Reply:
Feito